cinema_service/settings.py

```diff
@@ -20,9 +20,8 @@
 # See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = (
+SECRET_KEY =\
     "django-insecure-6vubhk2$++agnctay_4pxy_8cq)mosmn(*-#2b^v4cgsh-^!i3"
-)
 
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
```

Comment:
The `DEBUG` setting is currently set to `True`. This should be set to `False` in production to prevent exposing sensitive information. Consider using environment variables to toggle this setting based on the environment.